common/lib/api-client/middleware/request-include.js

```diff
@@ -19,7 +19,11 @@ module.exports = {
       include = include.sort().join(',')
     }
 
-    req.params.include = include
+    if (include) {
+      req.params.include = include
+    } else {
+      delete req.params.include
```

Comment:
Would it not be enough to just wrap in the if block?

Reply:
Nope. Because we need to remove the property if the value of include is falsy rather than just set it to undefined.